### PR TITLE
{bio}[GCC/8.3.0,GCCcore/8.3.0,system/system] lDDT v1.2, PSIPRED v4.02, CSBLAST v2.2.3

### PR DESCRIPTION
**Pull request by zemu-unile**
(created using `eb --new-pr`)
Standalone dependencies for RoseTTAFold

### CODE REVIEW

File: easybuild/easyconfigs/p/psipred/psipred-4.02-GCC-8.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+easyblock = 'MakeCp'
+
+name = 'psipred'
```
Comment (branfosj) [inline easybuild/easyconfigs/p/psipred/psipred-4.02-GCC-8.3.0.eb:3]: Please use `PSIPRED` for the name.

File: easybuild/easyconfigs/l/lddt/lddt-1.2.eb [hunk 2]
```diff
@@ -0,0 +1,23 @@
+easyblock = 'Tarball'
+
+name = 'lddt'
```
Comment (branfosj) [inline easybuild/easyconfigs/l/lddt/lddt-1.2.eb:3]: Please use `lDDT` for the name.

File: easybuild/easyconfigs/c/csblast/csblast-2.2.3-GCCcore-8.3.0.eb [hunk 3]
```diff
@@ -0,0 +1,37 @@
+easyblock = 'MakeCp'
+
+name = 'csblast'
```
Comment (branfosj) [inline easybuild/easyconfigs/c/csblast/csblast-2.2.3-GCCcore-8.3.0.eb:3]: Please use `CS-BLAST` for the name

File: easybuild/easyconfigs/l/lddt/lddt-1.2.eb [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+sources = ['%(name)s-linux.zip']
+checksums = ['3f4f72dc790f64431d29f7ff04353d3bbade8c8c12a487a1a99240a82fc4fcec']
+
+modextrapaths = {'PATH': ''}
+
+sanity_check_paths = {
+    'files': ['lddt', 'stereo_chemical_props.txt'],
+    'dirs': [],
+}
+
```
Comment (branfosj) [inline easybuild/easyconfigs/l/lddt/lddt-1.2.eb:22]: Especially for `Tarball` we should add a `sanity_check_command` to test the provided binary. I think we can use `lddt -e` here.

**Review by branfosj — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/l/lddt/lddt-1.2.eb (hunk 4 above)
Comment (zemu-unile) [reply easybuild/easyconfigs/l/lddt/lddt-1.2.eb]: Done, sadly this wont work for the other 2 as they provide no version check

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
@boegelbot please test @ generoso

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
bear-pg0211u08b.bear.cluster - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.8.5
See https://gist.github.com/bb7cd4e3b3034a1ef0fc5adeec1e0d68 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.3, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/262ff7d4627a811acfad36e7fc1651ed for a full test report.

**Comment by branfosj** [comment]
Going in, thanks @zemu-unile!

**Pull request merged**